addons/website/static/src/js/content/menu.js

```diff
@@ -233,4 +235,151 @@ publicWidget.registry.menuDirection = publicWidget.Widget.extend({
         $menu.addClass('dropdown-menu-' + alignment);
     },
 });
+
+publicWidget.registry.fixedHeader = publicWidget.Widget.extend({
+    selector: 'header #top_menu',
+
+    /**
+     * @override
+     */
+    start: function () {
+        self = this;
+        var menuLoading = $('o_menu_loading');
+        var header = $('.o_header_fixed');
+        if (header) {
+            if (!menuLoading) {
+                self._fixedHeader();
+            } else {
+                this.$el.one('fixedHeader', self._fixedHeader);
+            }
+        }
+        return this._super.apply(this, arguments);
+    },
+    /**
+     * @override
+     */
+    destroy: function () {
+        $('main').css('padding-top', '');
+        $('.o_header_fixed').removeClass('o_header_affix affixed');
+        this._super.apply(this, arguments);
+    },
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * @private
+     */
+    _fixedHeader: function () {
```

Comment:
A function name should start with a verb in most cases. Something like `_initializeFixedHeader` ?